packages/spatial/src/physics/components/TriggerComponent.ts

```diff
@@ -91,14 +91,14 @@ export const TriggerComponent = defineComponent({
     const colliderComponent = useComponent(entity, ColliderComponent)
 
     useEffect(() => {
-      if (!colliderComponent?.value) return
+      if (!colliderComponent?.value || !colliderComponent?.hasCollider.value) return
```

Comment:
we should just move this useEffect to the collider component

Reply:
implemented